doc/src/fix_shake.rst

```diff
@@ -27,9 +27,9 @@ Syntax
 
   .. parsed-literal::
 
-       *b* values = one or more bond types
-       *a* values = one or more angle types
-       *t* values = one or more atom types
+       *b* values = one or more bond types (may use typelabels)
```

Comment:
I think we have always used two words for 'type label'

Reply:
Ok. Made the change, also for fix shake docs.